karma.conf.js

```diff
@@ -15,6 +15,7 @@ module.exports = function (config) {
       require('@angular/cli/plugins/karma')
     ],
     files: [
+      {pattern: './node_modules/bootstrap/dist/css/bootstrap.min.css', included: true, watched: false},
```

Comment:
@IlyaSurmay does it works? or forgot to remove?

Reply:
It does. Without css result will be the same as it was before.